## Literature Cited

AIBS [American Institute of Biological Sciences]. (2015). Addressing Biological
    Informatics Workforce Needs: A Report from the 2015 AIBS Council of Member
    Societies and Organizations Meeting. Reston, Virginia. Retrieved from:
    https://www.aibs.org/public-policy/resources/AIBS_2015_Council_Report.pdf

Allen, J. M., Folk, R. A., Soltis, P. S., Soltis, D. E. and Guralnick, R. P. (2019a). Biodiversity synthesis across the green branches of the tree of life. Nature Plants, 5: 11–13. https://www.nature.com/articles/s41477-018-0322-7

Allen, J. M., Germain-Aubrey, C. C., Barve, N., Neubig, K. M., Majure, L. C., Laffan, S. W., Mishler, B. D., Owens, H. L., Smith, S. A., Whitten, W. M., Abbott, J. R., Soltis, D. E., Guralnick, R. and Soltis, P. S.. (2019b). Spatial phylogenetics of Florida vascular plants: the effects of calibration and uncertainty on diversity estimates. iScience 11: 57–70. https://doi.org/10.1016/j.isci.2018.12.002

Anselin, L., Syabri, I. and Kho, Y. (2006). GeoDa: an introduction to spatial data analysis. Geographical Analysis, 38(1): 5–22. https://doi.org/10.1111/j.0016-7363.2005.00671.x

Beck, K., Beedle, M., Bennekum, A., Cockburn, A., Cunningham, W., Fowler, M., Grenning, J., Highsmith, J., Hunt, A., Jeffries, R., Kern, J., Marick, B., Martin, R., Mellor, S., Schwaber, K., Sutherland, J. and Thomas, D. (2001). Manifesto for Agile Software Development. Retrieved from: http://www.agilemanifesto.org

Boyer, D. M., Gunnell, G. F., Kaufman, S. and McGeary, T. M. (2016). Morphosource: Archiving and sharing 3-d digital specimen data. The Paleontological Society Papers, 22: 157–181.

Cavner, J. A., Stewart, A. M., Grady, C. J. and Beach, J. H. (2012). An innovative Web Processing Services based GIS architecture for global biogeographic analyses of species distributions. OSGeo Journal 10: 15–25.

Chapman, A. D. (2005). Principles of Data Quality. Report for the Global Biodiversity Information Facility, Copenhagen. Version 1.0. Australian Biodiversity Information Services, Toowoomba South, Queensland, Australia. Retrieved from: https://www.gbif.org/document/80509/principles-of-data-quality 

Collins, T., Kearny, M. and Maddison, D. (2013). The Ideas Lab concept, assembling the Tree of Life, and AVAToL. PLOS Currents ToL, Mar 7, 2013. https://doi.org/10.1371/currents.tol.0fdb85e1619f313a2a5a2ec3d7a8df9e

Darriba, D., Flouri, T. and Stamatakis, A. (2018). The State of Software for Evolutionary Biology. Molecular Biology and Evolution, 35(5): 1037–1046. http://doi.org/10.1093/molbev/msy014

ED and NSF [U. S. Department of Education and U. S. National Science Foundation]. (2013). Common Guidelines for Education Research and Development. Retrieved from: https://www.nsf.gov/pubs/2013/nsf13126/nsf13126.pdf

Eiserhardt, W.L., Antonelli, A., Bennett, D.J., Botigué, L.R., Burleigh, J.G., Dodsworth, S., Enquist, B.J, Forest, F., Kim, J. T., Kozlov, A. M., Leitch, I. J., Maitner, B. S., Mirabab, S., Piel, W.H., Pérez‐Escobar, O.A., Pokorny, S., Rahbek, C., Sandel, B., Smith, S.A., Stamatakis, A., Vos, R.A., Warnow, T. and Baker, W.J. (2018). A roadmap for global synthesis of the plant tree of life. American Journal of Botany, 105(3): 614–622. http://doi.org/10.1002/ajb2.1041

Fegraus, E. H., Andelman, S., Jones, M. B. and Schildhauer, M. (2005). Maximizing the Value of Ecological Data with Structured Metadata: An Introduction to Ecological Metadata Language (EML) and Principles for Metadata Creation. Bulletin Ecological Society of America, 86(3): 158–168. https://doi.org/10.1890/0012-9623(2005)86[158:MTVOED]2.0.CO;2 

Folk, R. A., Stubbs, R. L., Mort, M. E., Cellinese, N., Allen, J. M., Soltis, P. S., Soltis, D. E. and Guralnick, R. P. (In revision). Rates of niche and phenotype evolution lag behind diversification in a temperate radiation. Proceedings of the National Academy of Sciences, in revision.

Folk, R. A., Visger, C. J., Soltis, P. S., Soltis, D. E. and Guralnick, R. (2018). Assessing ancestral niche suitability and geographic range dynamics as drivers of hybridization in Heuchera (Saxifragaceae). American Naturalist 192: 171–187.

Gotelli, N. J. and Entsminger, G. L. (2003). Swap algorithms in null model analysis. Ecology, 84(2): 532–535.

Grady, C., Biotaphy documentation. (2019). GitHub website. https://biotaphy.github.io/analyses

Grady, C. and Smith, S., BiotaPhy. (2019). GitHub Biotaphy analysis code repository. https://github.com/biotaphy

Grady, C. J., Stewart, A. M. and Beach, J. H. (2019). A parallel, fill-based algorithm for creating random presence-absence matrices. In preparation.

Graham, C. H., Elith, J., Hijmans, R. J., Guisan, A., Peterson, A. T. and Loiselle, B. A. (2008). The influence of spatial errors in species occurrence data used in distribution models. Journal of Applied Ecology, 45(1): 239–247. http://doi.org/10.1111/j.1365-2664.2007.01408.x

Hijmans, R. J., Cameron, S. E., Parra, J. L., Jones, P. G. and Jarvis, Al (2005). Very high resolution interpolated climate surfaces for global land areas. International Journal of Climatology, 25(15): 1965–1978.

Hinchliff, C. E., Smith, S. A., Allman, J. F., Burleigh, J. G., Chaudhary, R., Coghill, L. M., Crandall, K. A., Deng, J., Drew, B. T., Gazis, R., Gude, K., Hibbett, D. S., Katz, L. A., Laughinghouse, H. D., McTavish, E. J., Midorf, P .E., Owen, C. L., Ree, R. H., Rees, J. A., Soltis, D. E., Williams, T. and Cranston, K. A. (2015). Synthesis of phylogeny and taxonomy into a comprehensive tree of life. Proceedings of the National Academy of Sciences, 112(41): 12764012769. https://doi.org/10.1073/pnas.1423041112

IPCC [Intergovernmental Panel on Climate Change]. (2014). Climate Change 2014: Synthesis Report. Contribution of Working Groups I, II and III to the Fifth Assessment Report [Eds.] Pachauri, R.K. and Meyer, L. A. IPCC. Geneva, Switzerland.

Jetz, W., McPherson, J. M. and Guralnick, R. P. (2012a). Integrating biodiversity distribution knowledge: toward a global map of life. Trends Ecol. Evol., 27: 151–159.

Jetz, W., Thomas, G. H., Joy, J. B., Hartmann, K. and Mooers, A. O. (2012b). The global diversity of birds in space and time. Nature, 491(7424): 444.

Kacsuk, P. (2014). Science Gateways for Distributed Computing Infrastructures. Switzerland: Springer International Publishing. https://doi.org/10.1007/978-3-319-11268-8

Kadirvel, S. and Fortes, J. A. B. (2012). A Grey-box Approach to Performance Prediction in Map-Reduce based Platforms. 2012 21st International Conference on Computer Communications and Networks (ICCCN), Munich, Germany. https://doi.org/10.1109/ICCCN.2012.6289311

Kapadia, N.H., Fortes, J. A. B. and Brodley, C. E. (1999). Predictive Application-Performance Modeling in a Computational Grid Environment, Proceedings of the 8th IEEE International Symposium on High Performance Distributed Computing (pp. 47–54). Redondo Beach, CA, USA. https://doi.org/10.1109/HPDC.1999.805281

Kelling, S., Hochachka, W. M., Fink, D., Riedewald, M., Caruana, R., Ballard, G. and Hooker, G. (2009). Data-intensive science: a new paradigm for biodiversity studies. BioScience, 59(7): 613–620.

Laffan, S. W., Lubarsky, E. and Rosauer, D. F. (2010). Biodiverse, a tool for the spatial analysis of biological and related diversity. Ecography, 33(4): 643–647. http://doi.org/10.1111/j.1600-0587.2010.06237.x

Lawlor, B. and Walsh, P. (2015). Engineering bioinformatics: building reliability, performance and productivity into bioinformatics software. Bioengineered, 6(4):193–203. http://dx.doi.org/10.1080/21655979.2015.1050162

Lawrence, K. A., Zentner, M., Wilkins‐Diehr, N., Wernert, J. A., Pierce, M., Marru, S. and Michael, S. (2015). Science gateways today and tomorrow: positive perspectives of nearly 5000 members of the research community. Concurrency and Computation: Practice and Experience, 27(16): 4252–4268.

Lee, G. J. and Fortes, J. A. B. (2019). Improving Data-Analytics Performance Via Autonomic Control of Concurrency and Resource Units. ACM Transactions on Autonomous and Adaptive Systems. In press. 

Leibold, M. A., Economo, E. P. and Peres‐Neto, P. (2010). Metacommunity phylogenetics: separating the roles of environmental filters and historical biogeography. Ecology Letters 13(10): 1290–1299.

Lu, L. M., Mao, L. F., Yang, T., Ye, J. F., Liu, B., Li, H. L., Sun, M., Miller, J. T., Mathews, S., Hu, H. H., Niu, Y. T., Peng, D. X., Chen, Y. H., Smith, S. A., Chen, M., Xiang, K. L., Le, C. T., Dang, V. C., Lu, A. M., Soltis, P. S., Soltis, D. E., Li, J. H. and Chen, Z. D. (2018). Evolutionary history of the angiosperm flora of China. Nature, 554(7691): 234–238. https://doi.org/10.1038/nature25485

Marrinan, T., Aurisano, J., Nishimoto, A., Bharadwaj, K., Mateevitsi, V., Renambot, L., Long, L., Johnson, A. and Leigh, J. (2014). SAGE2: A New Approach for Data Intensive Collaboration Using Scalable Resolution Shared Displays. 10th IEEE International Conference on Collaborative Computing: Networking, Applications and Worksharing. https://doi.org/10.4108/icst.collaboratecom.2014.257337

Matsunaga, A. and Fortes, J. A. B. (2010). On the use of machine learning to predict the time and resources consumed by applications. 2010 10th IEEE/ACM International Symposium on Cluster, Cloud and Grid Computing (pp. 495–504). Melbourne, VIC, Australia. https://doi.org/10.1109/CCGRID.2010.98

Matsunaga, A., Thompson, A., Figueiredo, R. J., Germain-Aubrey, C. C., Collins, M., Beaman, R. S., MacFadden, B. J., Riccardi, G., Soltis, P. S., Page, L. M. and Fortes, J. A. B. (2013). A computational and storage-cloud for integration of biodiversity collections. 2013 IEEE 9th International Conference on e-Science (pp. 78–87). Beijing, China.

Michener, W. K. and Jones, M. B. (2012). Ecoinformatics: supporting ecology as a data-intensive science. Trends in Ecol. & Evol., 27(2): 85–93. https://doi.org/10.1016/j.tree.2011.11.016

Miller, J. T., Pirzl, R., Rosauer, D., Jolley-Rogers, G. and Varghese, T. (2018). Phylolink: phylogenetically-based profiling, visualisations and metrics for biodiversity. Bioinformatics, 30: 108. http://doi.org/10.1093/bioinformatics/bty792

Mishler, B. D., Knerr, N., González-Orozco, C. E., Thornhill, A. H., Laffan, S. W. and Miller, J. T. (2014). Phylogenetic measures of biodiversity and neo- and paleo-endemism in Australian Acacia. Nature Communications, 5(1): 4473. http://doi.org/10.1038/ncomms5473

Nee, S., May, R. M. and Harvey, P. H. (1994). The reconstructed evolutionary process. Philosophical Transactions of the Royal Society of London. Series B: Biological Sciences, 344(1309): 305–311.

Nelson, G., Paul, D., Riccardi, G. and Mast, A. (2012). Five task clusters that enable efficient and effective digitization of biological collections. ZooKeys, 209(1–2): 19–45. http://doi.org/10.3897/zookeys.209.3135

Nelson, G., Sweeney, P., Wallace, L. E., Rabeler, R. K., Allard, D., Brown, H., Carter, J. R., Denslow, M. W., Ellwood, E. R., Germain‐Aubrey, C. C., Gilbert, E., Gillespie, E., Goertzen, L. R., Legler, B., Marchant, D. B., Marsico, T. D., Morris, A. B., Murrell, Z., Nazaire, M., Neefus, C., Oberreiter, S., Paul, D., Ruhfel, B. R., Sasek, T., Shaw, J., Soltis., P. S., Watson, K., Weeks, A. and Mast, A. R. (2015). Digitization workflows for flat sheets and packets of plants, algae, and fungi. Applications in Plant Sciences, 3: 1500065. https://doi.org/10.3732/apps.1500065

NSF [U. S. National Science Foundation]. National Center for Science and Engineering Statistics. (2017). Women, Minorities, and Persons with Disabilities in Science and Engineering: 2017. NSF Special Report 17-310. Arlington, VA. Retrieved from: https://ncses.nsf.gov/pubs/nsf19304/

NSF [U. S. National Science Foundation]. (2019). Harnessing the Data Revolution (HDR) at NSF. Retrieved from: https://www.nsf.gov/cise/harnessingdata/

O’Meara, B. C. (2012). Evolutionary inferences from phylogenies: a review of methods. Ann. Rev. of Ecol., Evol. and Syst., 43: 267–285. https://doi.org/10.1146/annurev-ecolsys-110411-160331

Open API Initiative (2018). OpenAPI Specification. Retrieved from: https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.2.md

Papadopoulos, P. M., Katz, M. J. and Bruno, G. (2003). NPACI Rocks: Tools and techniques for easily deploying manageable linux clusters. Concurrency and Computation: Practice and Experience, Special Issue: Cluster 2001, 15(7–8): 707–725.

Peng, G. (2018). The State of Assessing Data Stewardship Maturity-An Overview. Data Science Journal, 17(7): 1–12. https://doi.org/10.5334/dsj-2018-007

Peres-Neto, P. and Pearse, W. (2017). Virtual Issue: Biogeography. Methods in Ecol. and Evol. Retrieved from: http://www.methodsinecologyandevolution.org/view/0/virtualIssues/biogeography_vi.html

Qin, J., Crowston, K. and Kirkland, A. (2017). Pursuing Best Performance in Research Data Management by Using the Capability Maturity Model and Rubrics. Journal of eScience Librarianship, 6(2): e1113. https://doi.org/10.7191/jeslib.2017.1113

Rabosky, D. L., Chang, J., Title, P. O., Cowman, P. F., Sallan, L., Friedman, M., Kaschner, K., Garilao, C., Near, T. J., Coll, M. and Alfaro, M. E. (2018). An inverse latitudinal gradient in speciation rate for marine fishes. Nature, 559: 392–395.

Ronquist, F. and Sanmartin, I. (2011). Phylogenetic Methods in Biogeography. Ann. Rev. of Ecol., Evol. and Syst., 42: 441–64.

Revell, L. J. (2012). phytools: an R package for phylogenetic comparative biology (and other things). Methods in Ecol. and Evol., 3(2): 217–223. http://doi.org/10.1111/j.2041-210X.2011.00169.x

Sanderson, J. G. and Pimm, S. L. (2015). Patterns in Nature: The Analysis of Species Co-occurrences. Univ. Chicago Press, Chicago, Ill.

SmartBear Software (2019). Swagger UI. https://github.com/swagger-api/swagger-ui/

Smith, S. A., Beaulieu, J. M. and Donoghue, M. J. (2009). Mega-phylogeny approach for comparative biology: an alternative to supertree and supermatrix approaches. BMC Evolutionary Biology, 9(1): 37. http://doi.org/10.1186/1471-2148-9-37

Smith, S. A. and Brown, J. W. (2018). Constructing a broadly inclusive seed plant phylogeny. Amer. J. Botany, 103(3): 302–314. https://doi.org/10.1002/ajb2.1019

Smith, S. A. and O'Meara, B. C. (2012). treePL: divergence time estimation using penalized likelihood for large phylogenies. Bioinformatics, 28(20): 2689–2690. http://doi.org/10.1093/bioinformatics/bts492

Smith, S. A., Moore M. J., Brown J. W. and Yang, Y. (2015). Analysis of phylogenomic datasets reveals conflict, concordance, and gene duplications with examples from animals and plants. BMC Evolutionary Biology, 15:150. https://doi.org/10.1186/s12862-015-0423-0

Soberón, J. and Cavner, J. (2015). Indices of Biodiversity Pattern Based on Presence-Absence Matrices: A GIS Implementation. Biodiversity Informatics, 10: 22–34. https://doi.org/10.17161/bi.v10i0.4801 

Soltis, D. E. and Soltis, P. S. (2016). Mobilizing and integrating big data in studies of spatial and phylogenetic patterns of biodiversity. Plant Diversity, 38 (2016) 264–270.

Stewart, A., Grady, C. and Anhalt, B., Lifemapper. (2019). GitHub software repository. https://github.com/lifemapper/

Stewart, A. and Grady, C., Lifemapper Documentation. (2019). GitHub. http://lifemapper.github.io/

Strona, G., Nappo, D., Boccacci, F., Fattorini, S. and San-Miguel-Ayanz, J. (2014). A fast and unbiased procedure to randomize ecological binary matrices with fixed row and column totals. Nature Communications 5, article 4114. https://www.nature.com/articles/ncomms5114

Treloar, A. (2014). The Research Data Alliance: globally co‐ordinated action against barriers to data publishing and sharing. Learned Publishing, 27(5): S9-S13. https://doi.org/10.1087/20140503

Venters, C. C., Capilla, R., Betz, S., Penzenstadler, B., Crick, T., Crouch, S., Nakagawa, E. Y., Becker, C. and Carrillo, C. (2018). Software sustainability: Research and practice from a software architecture viewpoint. Journal of Systems and Software, 138: 174–188. http://www.sciencedirect.com/science/article/pii/S0164121217303072

Wiens J. J., Pyron R. A., Moen, D. S. (2011). Phylogenetic origins of local-scale diversity patterns and the causes of Amazonian megadiversity. Ecology Letters, 14: 643–652.

Williams, N., Stewart, A., Papadopoulos, P. (2017). Virtualizing Lifemapper software infrastructure for biodiversity expedition. Concurrency and Computation: Practice and Experience, 29(13). https://doi.org/10.1002/cpe.4137

Yu, L., Moretti, C., Thrasher, A., Emrich, S., Judd, K. and Thain, D. (2010). Harnessing parallelism in multicore clusters with the all-pairs, wavefront, and makeflow abstractions. Cluster Computing, 13(3): 243–256.
